tests/Unit/ApparentHorizons/Test_ApparentHorizonFinder.cpp

```diff
@@ -0,0 +1,350 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <array>
+#include <cstddef>
+#include <pup.h>
+#include <vector>
+
+#include "ApparentHorizons/ComputeItems.hpp" // IWYU pragma: keep
+#include "ApparentHorizons/FastFlow.hpp"
+#include "ApparentHorizons/Strahlkorper.hpp"
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "DataStructures/DataBox/Prefixes.hpp"
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Domain/Block.hpp"
+#include "Domain/CoordinateMaps/CoordinateMap.hpp"
+#include "Domain/Domain.hpp"
+#include "Domain/DomainCreators/Shell.hpp"
+#include "Domain/ElementId.hpp"
+#include "Domain/ElementMap.hpp"
+#include "Domain/InitialElementIds.hpp"
+#include "Domain/LogicalCoordinates.hpp"
+#include "Domain/Mesh.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/Tags.hpp"
+#include "Informer/Tags.hpp"     // IWYU pragma: keep
+#include "Informer/Verbosity.hpp"
+#include "NumericalAlgorithms/Interpolation/AddTemporalIdsToInterpolationTarget.hpp"  // IWYU pragma: keep
```

Comment:
As a general question (not for this PR, I think it's present in a lot of features), do you think it would be better to package the set of includes intended as typical use for the public interface into one header (and then also add the appropriate flags to the IWYU settings so it doesn't whine about the rest)?
It may have the effect of sometimes including things not needed, but for things like this and executables, the include lists are starting to be a significant fraction of the overall file.... 
@nilsdeppe @wthrowe @fmahebert @kidder (or anyone else with opinions) thoughts? If it's desirable, I'm happy making an issue for it.

Reply:
no, that defeats the whole purpose of having header files